### ISSUE TITLE
do not allow --linker-status (and possibly --compiler-status) without other options

### ISSUE THREAD

**Issue by schveiguy** [dlang/dub#2100]
`dub dustmite --linker-status=1` is a useless command. I ran this on a project with missing linker symbols, and the result was a completely empty project (after 2 days). I mistakenly thought the linker error would be the same, not thinking that a completely empty project does not link.

dub should reject this command (or warn with user confirmation that the likely result is going to be an empty project). It should only allow this option if one provides a --linker-regex option as well.

In other words, the --linker-status option really should not be used except to verify the linker status code exactly.

Probably the same thing with --compiler-status.
